src/recipes/dto/get-recipe-response.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Ingredient, PreparationStep, Rating } from '@prisma/client';
+
+export class GetRecipeResponse {
+  @ApiProperty()
+  id: string;
+
+  @ApiProperty()
+  title: string;
+
+  @ApiProperty()
+  description: string;
+
+  @ApiProperty()
+  categoryId: string;
+
+  @ApiProperty()
+  preparationTime: string;
+
+  @ApiProperty()
+  isPublic: boolean;
+
+  @ApiProperty()
+  createdAt: Date;
+
+  @ApiProperty()
+  authorId: string;
+
+  @ApiProperty()
+  ingredients?: Ingredient[];
+
+  @ApiProperty()
+  preparationSteps?: PreparationStep[];
+
+  @ApiProperty()
+  ratings?: Rating[];
```

Comment:
i think it won't generate the correct docs. you probably need to pass proper type DTO and isArray prop set to true